services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
```

Comment:
Add a blank line between different import groups to follow PEP-8. Apply this change to the other places as well